projects/netgrif-components-core/src/lib/task-content/services/field-converter.service.ts

```diff
@@ -94,12 +94,17 @@ export class FieldConverterService {
                 return new ButtonField(item.stringId, item.name, item.behavior, item.value as number,
                     item.placeholder, item.description, item.layout, item.validations, item.component, item.parentTaskId);
             case FieldTypeResource.FILE:
+                const bytesSize = item.component?.properties?.maxSize;
+                const allowTypes: string[] = item.component?.properties?.allowTypes?.split(",");
                 return new FileField(item.stringId, item.name, item.behavior, item.value ? item.value : {},
-                    item.placeholder, item.description, item.layout, null, null, item.validations, item.component,
-                    item.parentTaskId);
+                    item.placeholder, item.description, item.layout, bytesSize !== undefined ? bytesSize : null,
+                    allowTypes?.length > 0 ? (allowTypes.length > 1 ? allowTypes as FileUploadMIMEType[] : allowTypes[0]) : null, item.validations, item.component, item.parentTaskId);
```

Comment:
The ``allowTypes?.length > 0 ? (allowTypes.length > 1 ? allowTypes as FileUploadMIMEType[] : allowTypes[0]) : null`` part may be extracted into separate function for better code readability.

Reply:
done